profile/README.md

```diff
@@ -194,9 +217,9 @@ The same logic applies to Sagas that are backed by streaming processors and stor
 
 Note that although you can use this extension to adjust MongoDB into an Event Store, we do not necessarily recommend this.
 During the lifecycle of this extension, we have noticed predicaments with how MongoDB behaves, causing the Event Store to act suboptimal from a performance perspective.
-We thus recommend either Axon Server (the highest level of performance for event storage) or an RDBMS of choice instead.
+We thus recommend either [Axon Server](#axon-server) (the highest level of performance for event storage) or an RDBMS of choice instead.
 
```

Comment:
I believe it is better to write event store in lowercase.

Reply:
Makes sense, will do!!